test/func_network.py

```diff
@@ -85,9 +85,21 @@ def network_pktdriver_mtcp(self, driver):
 $_tapdev = "tap0"
 """, timeout=30)
 
-    p.join()
-    sleep(5)
+    p.join(timeout=45)
+    if p.is_alive():
+        stderr.write("Timeout on join() for little_webserver - killing ")
+        stderr.flush()
+        p.kill()
+        p.join(timeout=5)
+        if p.is_alive():
```

Comment:
The doc suggests to check exitcode
to distinguish if join timed out or succeeded.
https://docs.python.org/3/library/multiprocessing.html#multiprocessing.Process.exitcode
But I suppose is_alive() is the same thing?

Reply:
Yes `p.is_alive()` is essentially `p.exitcode is None`